react/features/gifs/components/native/GifsMenu.js

```diff
@@ -1,5 +1,6 @@
+import { setServerUrl } from '@giphy/js-fetch-api';
```

Comment:
Is this also used by the RN SDK? AFAIK it uses their native SDKs under the hood.